src/SwedbankPay.Sdk/PaymentInstruments/Card/CardPaymentAbortPayment.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    public class CardPaymentAbortPayment
+    {
+        public string Operation = "Abort";
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)